daisy/resources.go

```diff
@@ -89,6 +90,27 @@ func (r *baseResourceRegistry) get(name string) (*resource, bool) {
 	return res, ok
 }
 
+func resourceExists(client compute.Client, url string) (bool, error) {
+	if !strings.HasPrefix(url, "projects/") {
+		return false, fmt.Errorf("partial GCE resource URL %q needs leading \"projects/PROJECT/\"", url)
+	}
+	switch {
```

Comment:
Not a fan of the switch case, but we can prob remove this once we move to resources/zonal resources.